godtools/push/action.yaml

```diff
@@ -57,8 +61,8 @@ runs:
       if: startsWith(github.ref, 'refs/tags')
       shell: bash
       run: |
-        [ "${{ inputs.tag-latest }}" = "true" ] && docker tag ${{ inputs.image }} ${{ inputs.image }}:${{ github.ref_name }};
-        godtools docker push --registry-id "${{ inputs.registries }}" ${{ inputs.image }}:${{ github.ref_name }}
+        [ "${{ inputs.tag-latest }}" = "true" ] && docker tag ${{ inputs.image }} ${{ inputs.image }}:${{ inputs.destination-tag }};
+        godtools docker push --registry-id "${{ inputs.registries }}" ${{ inputs.image }}:${{ inputs.destination-tag }}
```

Comment:
Can you elaborate more on the case? Which repository did require such change?

Reply:
We are building our own Jenkins [images](https://github.com/PiwikPRO/PPMS-Jenkins/blob/master/.github/workflows/main.yml#L45)